box-office.rb

```diff
@@ -1,5 +1,12 @@
 require 'sinatra'
+require 'httparty'
 
 get '/' do
   'Hello world!'
 end
+
+get '/movies/:title' do |title|
+  response = HTTParty.get("http://api.themoviedb.org/3/search/movie?query=#{title}&api_key=#{ENV['TMDB_API_KEY']}")
```

Comment:
Oh I thought we were searching by title, let me see if TMDb let's you query by IMDB id